ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: news_post
+title: "Ruby 3.3.0-preview2 릴리스"
+author: "naruse"
+translator: "shia"
+date: 2023-09-14 00:00:00 +0000
+lang: ko
+---
+
+{% assign release = site.data.releases | where: "version", "3.3.0-preview2" | first %}
+
+Ruby {{ release.version }} 릴리스를 알리게 되어 기쁩니다. Ruby 3.3은 RJIT으로 명명된 새로운 순수 Ruby JIT 컴파일러를 추가하고, 파서 생성기로 Lrama를 사용하며, 특히 YJIT에서 많은 성능 향상이 있습니다.
+
+## RJIT
+
+* 순수 Ruby JIT 컴파일러 RJIT을 도입하고 MJIT을 대체했습니다.
+  * RJIT은 Unix 플랫폼에서 x86\_64 아키텍처만 지원합니다.
+  * MJIT과 달리 런타임에 C 컴파일러가 필요하지 않습니다.
+* RJIT은 실험 목적으로 존재합니다.
+  * 프로덕션 환경에서는 YJIT을 계속 사용해야 합니다.
+* Ruby용 JIT 개발에 관심이 있다면 [RubyKaigi 2023 3일차에 있었던 k0kubun의 발표](https://rubykaigi.org/2023/presentations/k0kubun.html#day3)를 확인하세요.
+
+## Bison을 Lrama로 대체
+
+* Bison 대신 [Lrama LALR 파서 생성기](https://github.com/yui-knk/lrama)를 사용하도록 변경했습니다. [Feature #19637](https://bugs.ruby-lang.org/issues/19637)
+  * 관심이 있다면 [Ruby 파서의 미래 비전](https://rubykaigi.org/2023/presentations/spikeolaf.html)을 참고하세요.
+
+## YJIT
+
+* 3.2 버전 대비 성능 대폭 향상
```

Comment:
I think this is a summary of improvements. The preview1 article said "Significant performance improvements," and I think that can also be updated.

```suggestion
* 3.2 버전 대비 주요 성능 개선 사항
```

Reply:
Can you update preview1 article too?

Reply:
sure~  ebbb5224d

Reply:
Thanks!